### PR TITLE
[EDU-2990] feat: add plugin gpt ref

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added reference for Plugin Chat GPT in PT.

[EDU-2990]: https://aziontech.atlassian.net/browse/EDU-2990?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/devtools/plugin-chat-gpt/plugin-chat-gpt.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Plugin Azion ChatGPT
+description: Crie, edite e exclua instâncias na Plataforma Azion diretamente através do ChatGPT.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/plugin-chat-gpt/plugin-chat-gpt.mdx:3]:
```suggestion
description: Crie, edite e exclua instâncias na plataforma da Azion diretamente através do ChatGPT.
```

File: src/content/docs/pt-br/pages/devtools/plugin-chat-gpt/plugin-chat-gpt.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+meta_tags: chatgpt, openai, plugin
+namespace: docs_plugin_gpt
+menu_namespace: pluginGptMenu
+permalink: /documentacao/devtools/plugin-chatpgt/
+---
+
+import Button from '~/components/Button.astro'
+
+O **Plugin Azion ChatGPT** é uma ferramenta que permite criar, editar e excluir instâncias na plataforma Azion diretamente através do [ChatGPT](https://chat.openai.com/auth/login).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/plugin-chat-gpt/plugin-chat-gpt.mdx:12]:
```suggestion
O **Plugin Azion ChatGPT** é uma ferramenta que permite criar, editar e excluir instâncias diretamente na plataforma da Azion através do **ChatGPT**.
```

File: src/content/docs/pt-br/pages/devtools/plugin-chat-gpt/plugin-chat-gpt.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Ao abrir um novo chat no ChatGPT Plus (GPT-4), um menu suspenso para plugins estará disponível com a opção **Plugin store**. Ao entrar na loja, você pode procurar por `Azion` e instalá-lo.
+
+<Button href="/pt/documentacao/" text="ir para o guia de instalação do plugin Azion ChatGPT" variant="secondary"></Button>
+
+---
+
+## Uso
+
+Depois de instalar o **Plugin Azion ChatGPT** em sua conta, o uso do plugin funciona enviando prompts e solicitando ações ao AI. Você pode usá-lo normalmente como faria com o ChatGPT, solicitando prompts específicos relacionados à Azion.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/plugin-chat-gpt/plugin-chat-gpt.mdx:45]:
```suggestion
Depois de instalar o **Plugin Azion ChatGPT** em sua conta, o uso do plugin funciona enviando prompts e solicitando ações à AI. Você pode usá-lo normalmente como faria com o ChatGPT, solicitando prompts específicos relacionados à Azion.
```

**Review by MarianaAguilera — APPROVED** [review]
deixei um par de sugestões...

**Review by bru-andrade — APPROVED** [review]
Perfeito!

File: src/content/docs/pt-br/pages/devtools/plugin-chat-gpt/plugin-chat-gpt.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Depois de instalar o **Plugin Azion ChatGPT** em sua conta, o uso do plugin funciona enviando prompts e solicitando ações à AI. Você pode usá-lo normalmente como faria com o ChatGPT, solicitando prompts específicos relacionados à Azion.
+
+Você pode, por exemplo:
+
+- Listar instâncias existentes
+- Atualizar instâncias
+- Alterar nomes de instâncias
+- Excluir instâncias
+
+<Button href="/pt/documentacao/" text="veja mais exemplos de prompts" variant="secondary"></Button>
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/devtools/plugin-chat-gpt/plugin-chat-gpt.mdx:54]: Faltou contributors 😃

**Review by bru-andrade — APPROVED** [review]
Arrasou

**Pull request closed without merging**